content/applications/inventory_and_mrp/inventory/management/products/uom.rst

```diff
@@ -2,106 +2,111 @@
 
-In the :menuselection:`Inventory application --> Master Data --> Products`, open the
-product which you would like to change the purchase/sale unit of
-measure, and click on *Edit*.
+In the :menuselection:`Inventory application --> Master Data --> Products`, open the product which
+you would like to change the purchase/sale unit of measure, and click on *Edit*.
 
-In the *General Information* tab, you can select the *Unit of
-Measure* in which the product will be sold, which is also the unit in
-which internal transfers will take place. You can also select the
-*Purchase Unit of Measure*, which is the unit in which you purchase
-the product.
+In the *General Information* tab, you can select the *Unit of Measure* in which the product will
+be sold, which is also the unit in which internal transfers will take place. You can also select
+the *Purchase Unit of Measure*, which is the unit in which you purchase the product.
 
-.. image:: uom/uom_04.png
-    :align: center
+.. image:: uom/internal-uom-vs-purchase-uom.png
+   :align: center
+   :alt: Specify Units of Measure by use: internal transfer or when the product is purchased.
 
 Transfer from One Unit to Another
 =================================
 
 Buy in the Purchase UoM
 -----------------------
 
-In the *Purchase* application, *Create* a new request for quotation
-in which you include the product with the different *Units of Measure*
-and *Confirm* it.
+In the *Purchase* application, *Create* a new request for quotation in which you include the
+product with the different *Units of Measure* and *Confirm* it.
 
-.. image:: uom/uom_05.png
-    :align: center
+.. image:: uom/rfq-with-diff-uom.png
+   :align: center
+   :alt: A request for quotation illustrating different Units of Measure.
 
-On the automatically generated purchase orders, the UoM used is the Box
-of 6, meaning the Purchase UoM. You have of course the possibility to
-manually modify the UoM if necessary. When you enter the *Receipt*
-which is linked to the purchase order, you can observe that the 10 boxes
-of 6 units have been converted in 60 units. Indeed, the stock is managed
-in units.
+On the automatically generated purchase orders, the UoM used is the Box of 6, meaning the Purchase
+UoM. You have of course the possibility to manually modify the UoM if necessary. When you enter the
+*Receipt* which is linked to the purchase order, you can observe that the 10 boxes of 6 units have
+been converted in 60 units. Indeed, the stock is managed in units.
 
-.. image:: uom/uom_06.png
-    :align: center
+.. image:: uom/uom-conversion-on-receipt.png
+   :align: center
+   :alt: Units are converted on purchase order receipts.
 
 Replenishment
 -------------
 
-When doing a replenishment via the *Replenish* button on the product
-form, you have the possibility to use a different unit of measure.
+When doing a replenishment via the *Replenish* button on the product form, you have the possibility
```

Comment:
- Delete the two images `replenish-button-on-product.png` (Line 88) and `replenish-product-uom-settings.png` (Line 92)
- Replace the one sentence on Lines 85-86 `When doing a replenishment...use a different unit of measure.` with `A request for quotation for a product can also be generated directly from the product form using the :guilabel:'Replenish' button. After clicking :guilabel:'Replenish', a replenish assistant box pops up. The purchase unit of measure can be manually edited here if needed. Then, click :guilabel:'Confirm' to create the RFQ.` [line break] `Next, click the :guilabel:'Units Forecasted' smart button on the product form and scroll down to :menuselection:'Forecasted Inventory --> Requests for quotation'. Click on the RFQ reference number to open the draft RFQ. The purchase UoM can also be edited here if needed.`

(^ single quotation marks = backticks in my suggested change)